<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MedImgPlan</class>
 <widget class="qMRMLWidget" name="MedImgPlan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>1205</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="collapOtherModule">
     <property name="text">
      <string>Go to Other Modules</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushModuleRobCtrl">
        <property name="text">
         <string>Robot Ctrl</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushModuleTargetViz">
        <property name="text">
         <string>Target Viz</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushModuleFreeSurfer">
        <property name="text">
         <string>Free Surfer Importer</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="collapModelSelect">
     <property name="text">
      <string>Model Selection</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="labelMeshSelectorSkin">
        <property name="text">
         <string>Skin Model:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="comboMeshSelectorSkin">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLModelNode</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelMeshSelectorBrain">
        <property name="text">
         <string>Brain Model:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLNodeComboBox" name="comboMeshSelectorBrain">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLModelNode</string>
         </stringlist>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="collapRegistration">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Registration</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QPushButton" name="pushDigitize">
        <property name="text">
         <string>Start Auto-Digitize</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="pushUsePreviousRegistration">
        <property name="text">
         <string>Use Prev Registration</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pushDigPrev">
        <property name="text">
         <string>Use Prev Dig</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="pushDigHighlighted">
        <property name="text">
         <string>Digitize Highlighted</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="qSlicerSimpleMarkupsWidget" name="markupsRegistration">
        <property name="defaultNodeColor">
         <color>
          <red>0</red>
          <green>255</green>
          <blue>0</blue>
         </color>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="pushPlanLandmarks">
        <property name="text">
         <string>All Landmarks Planned</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="pushRegister">
        <property name="text">
         <string>Register</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="ctkCollapsibleGroupBox" name="collapTRE">
        <property name="title">
         <string>Check Target Registration Error</string>
        </property>
        <property name="collapsed">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="1" column="0">
          <widget class="ctkSliderWidget" name="sliderColorThresh">
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>20.000000000000000</double>
           </property>
           <property name="value">
            <double>4.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushStartTRE">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelColorThresh">
           <property name="text">
            <string>Color Change Threshold: (mm)</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="pushStopTRE">
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="pushDigPrevAndDigHilight">
        <property name="text">
         <string>Use Prev Dig
 And Dig Highlighted</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="labelRegistration">
        <property name="text">
         <string>Collect at least 3 landmarks.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="collapToolPosePlan">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Tool Pose Plan</string>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="labelToolPosePlanB">
        <property name="text">
         <string>Method B (3 points)</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QLabel" name="labelToolPosePlanC1">
        <property name="text">
         <string>1. Put 1 point as the contact point;</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QCheckBox" name="checkPlanBrain">
        <property name="text">
         <string>Plan on Brain?</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="labelToolPosePlanA">
        <property name="text">
         <string>Method A (4 points)</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QLabel" name="labelToolPosePlanC">
        <property name="text">
         <string>Method C (2 points)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="labelToolPosePlanA3">
        <property name="text">
         <string>3. The last of the 3 points should be the tail direction of the tool</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="2">
       <widget class="qSlicerSimpleMarkupsWidget" name="markupsToolPosePlan">
        <property name="defaultNodeColor">
         <color>
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="labelToolPosePlanA2">
        <property name="text">
         <string>2. Put 3 points (counterclockwise) as the contact plane;</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="2">
       <widget class="QLabel" name="labelToolPosePlanC2">
        <property name="text">
         <string>2. Put another point as the tail direction of the tool</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0" colspan="2">
       <widget class="QPushButton" name="pushToolPosePlan">
        <property name="text">
         <string>Plan Coil Pose</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="labelToolPosePlanA1">
        <property name="text">
         <string>1. Put 1 point as the contact point center;</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="labelToolPosePlanB1">
        <property name="text">
         <string>1. Put 3 points (couterclockwise) as the contact plane;</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QLabel" name="labelToolPosePlanB2">
        <property name="text">
         <string>2. The last of the 3 points should be the tail direction of the tool</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0" colspan="2">
       <widget class="ctkCollapsibleGroupBox" name="collapGridPlan">
        <property name="title">
         <string>Grid Plan</string>
        </property>
        <property name="collapsed">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="6" column="0" colspan="2">
          <widget class="ctkSliderWidget" name="sliderGridDistanceApart">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="pageStep">
            <double>1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>8.000000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="labelGridDistanceApart">
           <property name="text">
            <string>Distance Apart: (mm)</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="ctkSliderWidget" name="sliderGridPlanNum">
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>144.000000000000000</double>
           </property>
           <property name="value">
            <double>16.000000000000000</double>
           </property>
           <property name="tickInterval">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="2">
          <widget class="QPushButton" name="pushPlanGrid">
           <property name="text">
            <string>Plan Grid</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelGridPlanNum">
           <property name="text">
            <string>Number of Targets:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0" colspan="2">
          <widget class="QPushButton" name="pushGridSetNext">
           <property name="text">
            <string>Set Next as Target</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QCheckBox" name="checkBoxGridAnatomySurf">
           <property name="text">
            <string>Plan on Anatomy Surface</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QCheckBox" name="checkBoxGridPerspPlane">
           <property name="text">
            <string>Plan on Perspective Plane</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerSimpleMarkupsWidget</class>
   <extends>qSlicerWidget</extends>
   <header>qSlicerSimpleMarkupsWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>MedImgPlan</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>markupsRegistration</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MedImgPlan</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>markupsToolPosePlan</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>580</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MedImgPlan</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>comboMeshSelectorSkin</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MedImgPlan</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>comboMeshSelectorBrain</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
